src/main/java/org/assertj/core/api/AbstractStringAssert.java

```diff
@@ -242,6 +243,31 @@ public SELF isValidBase64() {
     return myself;
   }
 
+  /**
+   * Decodes the actual value as a Base64 encoded string, the decoded value becoming the new object under test.
```

Comment:
`the decoded value becoming ` -> `the decoded value (a byte[]) becoming `

Reply:
What about `the decoded bytes becoming the new array under test`?